operating/default-security-policies.adoc

```diff
@@ -9,6 +9,15 @@ toc::[]
 [role="_abstract"]
 The default security policies in {product-title} provide broad coverage to identify security issues and ensure best practices for security in your environment. By configuring those policies, you can automatically prevent high-risk service deployments in your environment and respond to runtime security incidents.
 
+[NOTE]
+====
+The severity levels for policies in {product-title} are different from the severity levels that are assigned by Red Hat Product Security.
```

Comment:
```suggestion
The severity levels for policies in {product-title} are different from the severity levels that Red Hat Product Security assigns.
```
Rewording suggestion to avoid passive voice.